sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -139,7 +139,7 @@ export class Receiver {
           return;
```

Comment:
This wasn't handled before either, but if we get to this point and no receiver exists, should we trigger the error handler with an error to inform the user that an issue occurred? Looking at the code I don't think it's really possible to encounter this (assume we were keeping TypeScript happy) so I wouldn't block on it.

Reply:
You are right, it is not possible to encounter this.

If the promise returned by `StreamingReceiver.create` has resolved, then it means that the receiver was created. So, this check is not needed. 

Pushed a commit to remove it.

cc @ramya0820 as this check was added as part of your PR #2139

Reply:
did something change? 
iirc, it was to address TS compiler mostly

Reply:
nothing changed.